STM32/USBprint/Inc/usb_cdc_fops.h

```diff
@@ -17,22 +17,42 @@
 #ifndef USB_CDC_FOPS_H
 #define USB_CDC_FOPS_H
 
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "usbd_cdc.h"
+
 #ifdef __cplusplus
- extern "C" {
+    extern "C" {
 #endif
 
-#include "usbd_cdc.h"
-#include "stdbool.h"
+/***************************************************************************************************
+** DEFINES
+***************************************************************************************************/
+
+#define CIRCULAR_BUFFER_SIZE 1024
+
+#define CDC_ERROR_NONE              0x00000000U
```

Comment:
CDC_ERROR_NONE does not seem to be used anywhere.

Reply:
Good catch - it is supposed to be